docs/docsource/conf.py

```diff
@@ -102,14 +102,15 @@
 pygments_style = 'sphinx'
 
 
-# -- rst prolog  ---------------------------------------------------------
+# -- rst prolog ---------------------------------------------------------
+
 # This section contains text substitution for custom variables and link
-# substitution for MSFT doc links. As we become
-# more complete in our documentation, we can replace these with references
-# to our own documentation.
+# substitution for MSFT doc links. As we become more complete in our
+# documentation, we can replace these with references to our own documentation.
 
 rst_prolog = """
 
+   .. |product| replace:: XDM
```

Comment:
does this replace `|product|` with `XDM`? If so, wouldn't we want to use `|product|` as much as possible instead of XDM? Or is this just a safety net of sorts to replace them when they are around but not require them?

Reply:
IIRC this was a 1:1 swap for an existing structure. My job here was simply to replace, not to think too hard about it.